src/Helpers/TableToolbarHelper.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import messages from '../Messages';
+import { FILTERS } from './constants';
+import { filterCategories } from './constants';
+import { Button } from '@patternfly/react-core';
+import { intl } from '../Utilities/IntlProvider';
+
+export const handleChangePage = (_event, page, apply) => apply({ page });
+
+export const handleSetPageSize = (_event, perPage, apply) => apply({ page_size: perPage, page: 1 });
+
+export const handleFilter = methods => filterCategories.map(filter => {
```

Comment:
what's the purpose of this function, it's not called anywhere

Reply:
I forgot to delete :))